src/main/kotlin/edu/ics499/team1/app/entities/CompletedMaintenanceEntity.kt

```diff
@@ -0,0 +1,21 @@
+package edu.ics499.team1.app.entities
+
+import jakarta.persistence.*
+
+@Entity
+@Table(name = "completed_maintenance")
+data class CompletedMaintenanceEntity(
+    @Id
+    val maintenanceId: Int,
+    val name: String,
+    val notes: String?,
+    val pictures: String?,
+    val date: String,
+    val mileage: Int,
+    val serviceCenter: String?,
+    val mechanics: String?,
+    val totalCost: Double?,
+    @ManyToOne
+    @JoinColumn(name = "vehicle_id")
```

Comment:
Sets a ManyToOne relationship for the DB. I think of this as - Many is referring to the current object and One is referring to the object reference. So we can have Many completed maint. and they would only refer to One Vehicle.

Reply:
Join column, I am not 100% sure what this does exactly but I know it allows us to change the name of that column in the DB. I attempted @column(name = "vehicle_id") and that was failing so I found this. I am sure there is more to it though.

Anyway, without the name would be vehicle vehicle id